engine/src/main/java/com/redhat/cloud/notifications/events/HttpErrorType.java

```diff
@@ -1,6 +1,7 @@
 package com.redhat.cloud.notifications.events;
 
 public enum HttpErrorType {
+    SOCKET_TIMEOUT("a timeout happened while waiting for server response"),
```

Comment:
Small change to be consistent with other messages:
```suggestion
    SOCKET_TIMEOUT("a timeout happened while waiting for the HTTP server response"),
```